phpcs.xml

```diff
@@ -16,5 +16,14 @@
     <file>test</file>
 
     <!-- Include all rules from the Laminas Coding Standard -->
-    <rule ref="LaminasCodingStandard"/>
+    <rule ref="LaminasCodingStandard">
+        <!-- Sniff cannot handle multiline array-shape parameter declarations -->
+        <exclude name="Squiz.Commenting.FunctionComment.MissingParamName"/>
+        <!--
+            Sniff wants to enforce spaces after each comma but does not allow spaces before comma.
+            Due to the usage of list() pattern along with skipped arguments, i.e. list($foo,,$bar), there is no
+            way to fulfill this without introducing an unused variable, which then would be reported by psalm.
```

Comment:
```php
list($foo,,$bar) // invalid due to missing space after first and second `,`
list($foo, , $bar) // invalid due to additional whitespace before second `,`
list($foo, $baz, $bar) // valid but reported by psalm due to unused variable
```

No clue if this should be reported upstream or ignored per-file or whatever. I've added this here for now as I do not want to hassle with this stuff. 

----

I am +1 on removing codestyle and implementing `php-cs-fixer`. Stuff which cannot be fixed automatically should not get reported at all. But thats a whole another topic.